randovania/games/factorio/generator/item_cost.py

```diff
@@ -0,0 +1,155 @@
+from __future__ import annotations
+
+import dataclasses
+import math
+
+from randovania.games.factorio.data_importer import data_parser
+
+BASIC_RESOURCES = {"water", "steam", "crude-oil", "iron-ore", "copper-ore", "coal", "stone"}
+_k_fluids = {"water", "steam", "crude-oil", "heavy-oil", "light-oil", "petroleum-gas", "lubricant", "sulfuric-acid"}
```

Comment:
what's the `k` mean? and shouldn't this be allcaps?

Reply:
Constant, which makes it really funny with the other convention for constants one line above